plugins/system/skeletonkey/media/js/backend.js

```diff
@@ -50,7 +50,7 @@ const initSkeletonKey = () => {
 		// Create the login button itseld
 		const elLink = document.createElement('button');
 		elLink.setAttribute('type', 'button');
-		elLink.classList.add('btn', 'btn-dark', 'btn-sm');
+		elLink.classList.add('btn', 'btn-dark', 'btn-sm', 'ms-1');
```

Comment:
This is wrong. It's a button _group_. That's why the buttons appear run together. You will also observe that the rounded corners are also applied at left side of the leftmost button and right hand side of the rightmost button. There must be no margin between the buttons, so I am removing that.